client/components/OodlerEditTile.vue

```diff
@@ -0,0 +1,176 @@
+            </div>
+          </div>
+          <table v-show="!editing.lastName" class="table table-striped table-hover col-md-12">
+            <tbody>
+            <tr @click="edit('lastName')">
+              <td class="col-lg-2">Last Name</td>
+              <td class="col-lg-8">{{ oodler.lastName }}</td>
+              <td class="col-lg-2"><a class="hover-btn"><span class="glyphicon glyphicon-pencil"></span></a></td>
+            </tr>
+            </tbody>
+          </table>
+        </form>
+  
+        <form @submit="updateOodler('email')" class="form-horizontal col-md-12">
+          <div v-show="editing.email" class="form-group form-group-sm">
+            <label for="email" class="col-lg-2 control-label">Email</label>
+            <div class="col-lg-6">
+              <input type="text" v-model="editOodler.email" class="form-control input-sm" id="email">
+            </div>
+            <div class="btn-group col-lg-4">
+              <button type="submit" class="col-lg-8 btn btn-sm btn-success">Save</button>
+              <button type="reset" @click="cancel('email')" class="col-lg-4 btn btn-sm btn-default"><span class="glyphicon glyphicon-remove"></span></button>
+            </div>
+          </div>
+          <table v-show="!editing.email" class="table table-striped table-hover col-md-12">
+            <tbody>
+            <tr @click="edit('email')">
+              <td class="col-lg-2">Email</td>
+              <td class="col-lg-8">{{ oodler.email }}</td>
+              <td class="col-lg-2"><a class="hover-btn"><span class="glyphicon glyphicon-pencil"></span></a></td>
+            </tr>
+            </tbody>
+          </table>
+        </form>
+  
+        <form @submit="updateOodler('office')" class="form-horizontal col-md-12">
+          <div v-show="editing.office" class="form-group form-group-sm">
+            <label for="office" class="col-lg-2 control-label">Office</label>
+            <div class="col-lg-6">
+              <input type="text" v-model="editOodler.office" class="form-control input-sm" id="office">
+            </div>
+            <div class="btn-group col-lg-4">
+              <button type="submit" class="col-lg-8 btn btn-sm btn-success">Save</button>
+              <button type="reset" @click="cancel('office')" class="col-lg-4 btn btn-sm btn-default"><span class="glyphicon glyphicon-remove"></span></button>
+            </div>
+          </div>
+          <table v-show="!editing.office" class="table table-striped table-hover col-md-12">
+            <tbody>
+            <tr @click="edit('office')">
+              <td class="col-lg-2">office</td>
+              <td class="col-lg-8">{{ oodler.office }}</td>
+              <td class="col-lg-2"><a class="hover-btn"><span class="glyphicon glyphicon-pencil"></span></a></td>
+            </tr>
+            </tbody>
+          </table>
+        </form>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+  
+  export default {
+    data() {
+      return {
```

Comment:
For `editOodler` I agree: `editedOodler` would be better (I already have var `oodler` as a prop).
But for `editing` object it's not action, it's a state (true or false i.e. is firstName's state editing or not?) Why is that wrong approach?